### ISSUE TITLE
[Amazon VOD] Decoding video stream for movies fails

### ISSUE THREAD

**Issue by Clubfan22** [Sandmann79/xbmc#217]
Hi, first an foremost, thank you for your great work on the Amazon VOD addon!

When I try to play a movie, no video is being played while the sound works with no problems.
Interestingly, playing an episode of a TV series is working flawlessly, regardless of resolution.
You can find the log [here](https://gist.github.com/Clubfan22/7b1542ed52792f44e07f86cd311a6860).

Because I have not tried to watch a movie for the last month or two, I can sadly not tell whether it is related to a client-side update. If this problem is not reproducible for others, I can try bisecting it.

This issue might be related to an [issue of inputstream-adaptive](https://github.com/peak3d/inputstream.adaptive/issues/212) as the error in the log is the same:
`ERROR: AddOnLog: InputStream Adaptive: DecodeVideo: kNoKey for key 1F5146528EE64BB5AE0106F751A16351`

I'm using a german Amazon Prime account and the following configuration:

Arch Linux
Kodi 18.0-RC3 Git: 20181214-4c17e7769e5 (built with -O3 and -march=native, installed through AUR)
Amazon VOD Version 0.6.3
inputstream.adaptive 2.3.12 (installed through AUR)
libvidewinecdm.so (installed through the helper script)

I'd be more than happy to provide additional information if you would like me to.

**Comment by Clubfan22** [comment]
Further research shows that video playback works correctly when I change the following settings of InputStream-Adaptive:

Max. Resolution general decoder: 1080p -> 480p
Max. Resolution secure decoder: 1080p -> 480p

**Comment by Varstahl** [comment]
As far as I looked last time I checked (several months back), high resolution movies are encoded in a different Widevine encryption than the lower resolution movies and the tv series. As far as I know, it's a Widevine/IS.A problem, which we can't fix.

I'm not aware of any recent changes to Kodi or IS.A that could/would break the addon in itself. I'm using a one month-ish old build and latest IS.A, and it still reaches the 480p-540p by itself without failing to decode.

**Comment by mirh** [comment]
480p is exactly the maximum WV L3 supports. 
This means this is just about L1 not working, which I would quite much guess has to do with: https://github.com/peak3d/inputstream.adaptive/issues/185

**Comment by Varstahl** [comment]
Nah, it's not the addon: `[swscaler] No accelerated colorspace conversion found from yuv420p to bgra`. As for the other issue, there are a number of things that could go wrong, but as soon as the video plays it's not the addon, and I can't help.

**Comment by Clubfan22** [comment]
Indeed, installing an older Widevine version from http://mirror.yandex.ru/fedora/russianfedora/russianfedora/nonfree/el/updates/7/x86_64//chromium-widevine-plugin-1.4.9.1088-3.el7.R.x86_64.rpm fixed it for me.
